iotilebuild/test/test_iotilebuild/python_data_bundling/SConstruct

```diff
@@ -0,0 +1,3 @@
+import autobuild
```

Comment:
I see that you added a component with data files but I don't see any actual test case verifying that it builds correctly?

Reply:
Fixed. Added test case.

Reply:
Fixed. I don't know if you have a preference, but I added the test case to `test_scons.py`. I saw that a number of test components were doing that, and some had their tests in their individual `python/test`.